### ISSUE TITLE
Allow *SOME* cheat modes to be enabled permanently in ini/registry

### ISSUE THREAD

**Issue by pbatard** [pbatard/rufus#436]
Not sure if things that are voluntarily disabled such as listing USB HDDs should be part of that enhancement, but stuff like size units, VMWare VMHD listing or the upcoming timestamp preservation (#389), should probably be stored on permanent basis.

**Comment by maxxc0m** [comment]
Self-assign

**Comment by pbatard** [comment]
> Self-assign

???

**Comment by MagicAndre1981** [comment]
this is a SPAM account.

But the option to always show USB HDDs would be nice.

**Comment by pbatard** [comment]
For some. On the other hand, what wouldn't be nice is the angry reports I am virtually guaranteed to get from people who got that option enabled by default, and ended up formatting an external USB HDD with important data because they weren't paying attention, but will tell me how Rufus should have done much more to prevent that.

Heck, even with USB HDD detection never enabled by default, I got multiple complaints from people indicating that Rufus should have done more babysitting to prevent their data loss, when they picked up the wrong target. I did some of that by adding extra warnings when multiple partitions are detected on the target, but it never seems enough.

I'm afraid therefore that your convenience will have to come after my goal of preventing data loss. For that specific reason, besides the `-f` commandline switch (because it's difficult to leave that one enabled by mistake), I am not currently planning to have a setting that allows USB HDDs to be detected by default...
